src/util.ts

```diff
@@ -172,9 +172,9 @@ export function truthy(arg: AnyVal): boolean {
 export function isEmpty(x: AnyVal): boolean {
```

Comment:
isEmpty will genereate different result compare to isEmpty function of  lodash.
 _.isEmpty(0) === true

Reply:
I missed this comment earlier but regardless I think we should prefer our current semantics. It is also for such discrepancies that `mingo` has no dependencies.  `0` being empty overloads the meaning of empty, although a case can be made for it. What about `false`, is that also empty. If not empty then why? 

Our current semantics make few assumptions which makes things easier to reason about IMO.